CalendarApp/BookingRoom/Models/GoogleEvent/ToEventConverter.cs

```diff
@@ -9,14 +10,17 @@ public static Event ToEvent(CalendarEvent eventForConverting)
         {
             Event newEvent = new Event
             {
+                Recurrence = eventForConverting.Recurrence,
                 Summary = eventForConverting.Summary,
                 Description = eventForConverting.Description,
                 Start = new EventDateTime
                 {
+                    TimeZone = "Europe/Kiev",
```

Comment:
That is something that you should always avoid. You need to get timezone from calendar or get current timezone